Source/Core/AudioCommon/Mixer.h

```diff
@@ -101,9 +101,14 @@ class Mixer final
     {
     public:
       constexpr Granule() = default;
-      constexpr Granule(const GranuleBuffer& input, std::size_t start_index);
+      constexpr Granule(const Granule&) = delete;
+      constexpr Granule& operator=(const Granule&) = delete;
 
-      static StereoPair InterpStereoPair(const Granule& front, const Granule& back, u32 frac);
+      void Reset();
+      void Set(const GranuleBuffer& input, const std::size_t start_index);
+      void Set(const Granule& input, const StereoPair fade);
```

Comment:
I really have no understanding of the math going on with the `GRANULE_WINDOW` but these two `Set` functions seem to do very different things. Are better/different names possible?